services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/utils/MongoDbDocumentBuilder.java

```diff
@@ -46,6 +48,22 @@ public MongoDbDocumentBuilder withDeviceId(final String deviceId) {
         return this;
     }
 
+    /**
+     * Adds nested CA subjectDn property.
+     *
+     * @param entityPropKey key where entity properties are stored. e.g.: "tenant" for
+     *                      the {@link org.eclipse.hono.service.management.tenant.Tenant}
+     * @param caSubjectDn certificate Authority subjectDn
```

Comment:
How about renaming the parameter to `subjectDN` instead of `caSubjectDn`?
Also `@param The subject DN. `

Reply:
done

Reply:
How about also updating to`@param The subject DN` as in the above comment.

Reply:
Any update on this?

Reply:
done